form_components/AppointmentForm.js

```diff
@@ -71,13 +73,21 @@ const styles = StyleSheet.create({
         marginBottom: 5,
         fontSize: 18,
         borderRadius: 6,
-        backgroundColor:'#EDEFEF'
+        backgroundColor:'#EDEFEF',
+        width: 400,
+        alignSelf: 'center'
     },
     addButton:{
         backgroundColor:'white',
         padding:10,
         width:200,
         left:100
+    },
+    formName: {
```

Comment:
This style could also be added to GlobalStyle since we use it in at least 2 files. Would help us keep our code DRY.